src/frontend/src/App.vue

```diff
@@ -1,13 +1,102 @@
 <template>
   <div id="app">
+    {{ mining }}
     <router-view />
   </div>
 </template>
 
 <script lang="ts">
 import { Vue } from "vue-property-decorator";
+import { io } from "socket.io-client";
+import { sha256 } from "js-sha256";
+import type { Transaction } from "./types";
 
-export default class App extends Vue {}
+const socket = io("http://localhost");
+
+
+
+export default class App extends Vue {
+
+  get walletId(){
+    return this.$store.getters.walletId
+  }
+
+  get findProof(){
+    return this.$store.getters.findProof
+  }
+
+  validHash (hash: string): boolean  {
+      return hash.startsWith("0000", 0);
+  }
+
+  computeHash(nonce: number, transaction: Transaction): string {
+
+      const toHash = (nonce.toString()+transaction.amount.toString()+transaction.id+transaction.signature).replace(/(\r\n|\n|\r)/gm, "");
```

Comment:
Could you maybe indent this or add space between pluses so it is easier to read please!